systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -646,7 +646,6 @@ void testCertRenewalInMaintenanceWindow(ExtensionContext extensionContext) {
         final String namespaceName = StUtils.getNamespaceBasedOnRbac(NAMESPACE, extensionContext);
         final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
         final String topicName = mapWithTestTopics.get(extensionContext.getDisplayName());
-        final String userName = mapWithTestUsers.get(extensionContext.getDisplayName());
```

Comment:
so you can add it to the user instead of the `clusterName`

Reply:
Sorry, I don't understand this one.

Reply:
Done, resolved.